app/views/shared/_location_map.html.erb

```diff
@@ -20,6 +20,10 @@
   <% if locals[:geojson_field].present?  %>
     drawMode="true"
   <% end %>
+  <% if @blank_layout %>
+    hideResetControl="true"
+    staticMode="true"
+  <% end %>
```

Comment:
We don't want to show those interactive controls on the pdf version so these are the required props to switch that off